src/tools/collapse_dataset_tool.js

```diff
@@ -88,9 +91,9 @@ morpheus.CollapseDatasetTool.prototype = {
       set.remove(field);
     });
     // hide fields that were not part of collapse to
-    set.forEach(function (val, name) {
-      heatMap.setTrackVisible(name, false, !rows);
-    });
+    // set.forEach(function (val, name) {
```

Comment:
There is no need anymore for hiding unused fields for collapsing, because in old tab they should remain unchanged and on the new tab, they won't be shown anyway.